admin/cmd/main.go

```diff
@@ -29,11 +29,15 @@ type Config struct {
 
 func main() {
 	// Load .env (note: fails silently if .env has errors)
-	godotenv.Load()
+	err := godotenv.Load()
+	if err != nil {
+		fmt.Printf("Failed to load godotenv: %s", err.Error())
+		os.Exit(1)
+	}
```

Comment:
We actually want to ignore the error here. Maybe do `_ := godotenv.Load()`?

Reply:
Yeah `_ := godotenv.Load()` works